ecomap/ecomap/AddProblem/Controllers/AddProblemViewController.m

```diff
@@ -0,0 +1,291 @@
+//
+//  AddProblemViewController.m
+//  ecomap
+//
+//  Created by Anton Kovernik on 10.02.15.
+//  Copyright (c) 2015 SoftServe. All rights reserved.
+//
+
+#import "AddProblemViewController.h"
+
+@interface AddProblemViewController () {
+    CGFloat padding;
+    CGFloat paddingWithNavigationView;
+    CGFloat screenWidth;
+}
+
+//AddProblemProperties
+@property (weak, nonatomic) IBOutlet UIButton *nextButton;
+@property (weak, nonatomic) IBOutlet UIButton *prevButton;
+@property (weak, nonatomic) IBOutlet UIPageControl *pageControl;
+@property (weak, nonatomic) IBOutlet UIPickerView *problemTypesPickerVIew;
+@property (weak, nonatomic) IBOutlet UIButton *addProblemButton;
+
+@property (strong, nonatomic) UIBarButtonItem *closeButton;
+@property (nonatomic, strong) NSArray *problemTypes;
+@property (nonatomic) BOOL isNextButtonTaped;
+
+//AddProblemViews
+@property (nonatomic, strong) UIView* addProblemNavigationView;
+@property (nonatomic, strong) UIView* addProblemLocationView;
+@property (nonatomic, strong) UIView* addProblemNameView;
+@property (nonatomic, strong) UIView* addProblemTypeView;
+@property (nonatomic, strong) UIView* addProblemDescriptionView;
+@property (nonatomic, strong) UIView* addProblemSolutionView;
+@property (nonatomic, strong) UIView* addProblemPhotoView;
+@property (nonatomic, strong) UIView* curView;
+@property (nonatomic, strong) UIView* prevView;
+@property (nonatomic, strong) UIView* nextView;
+
+
+
+
+@end
+
+@implementation AddProblemViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    [[NSNotificationCenter defaultCenter] addObserver:self  selector:@selector(orientationChanged:)
+                                                 name:UIDeviceOrientationDidChangeNotification
+                                               object:nil];
+}
+
+#pragma mark - PickerView
+
+- (NSString *)pickerView:(UIPickerView *)pickerView titleForRow:(NSInteger)row forComponent:(NSInteger)component {
+    return _problemTypes[row];
```

Comment:
General comment for entire project: please, use one style for property access. It looks not good if in one place you've used invar access _problemTypes, in other place self.problemTypes, in other you have backup by inverse, in other don't. That is a bad style. Please, choose one. I will recommend to use properties and self accessor syntax.